MvvmCross/ViewModels/MvxAppStart.cs

```diff
@@ -14,21 +16,31 @@ public class MvxAppStart<TViewModel>
     {
         protected readonly IMvxNavigationService NavigationService;
 
+        private int startHasCommenced;
+
         public MvxAppStart(IMvxNavigationService navigationService)
         {
             NavigationService = navigationService;
         }
 
-        public async void Start(object hint = null)
+
+
+        public void Start(object hint = null)
         {
+            // Check whether Start has commenced, and return if it has
+            if (Interlocked.CompareExchange(ref startHasCommenced, 1, 0) == 1) return;
+
             if (hint != null) {
                 MvxLog.Instance.Trace("Hint ignored in default MvxAppStart");
             }
+            
             try {
-                await NavigationService.Navigate<TViewModel>();
+                NavigationService.Navigate<TViewModel>().Wait();
```

Comment:
I think using GetAwaiter().GetResult() is prefered over .Wait()

Reply:
Done